void myFunction(String test)
throw new Exception("Test");